examples/aks-production/cluster.lokocfg

```diff
@@ -33,21 +31,6 @@ variable "location" {
   default = "West Europe"
 }
 
-variable "state_s3_key" {
```

Comment:
Why do we remove it? For production setups using local state is not recommended. And we do not have an alternative right now. See #361.

Reply:
Ok, I did not know about it, will revert it back

Reply:
cc @surajssd

Reply:
I don't think it is practical for someone to use S3 when they are deploying cluster to AKS.

It is more practical to store the assets locally than someone creating an account on AWS just for storing assets on S3. I think we should not add S3 or any AWS related stuff for Azure. It is not how production setup for a pragmatic user will look like.

Reply:
Either we add a way to store assets on Azure and then add that to this docs OR use local dir to store assets.

Reply:
I disagree. If we cannot guarantee production grade on a given platform, then I would remove this config alltogether and leave testing only.

Reply:
Do you mean AKS is not production ready?

Reply:
>Do you mean AKS is not production ready?

No, I mean "AKS configuration without state in remote storage" is in my opinion not production ready.

Reply:
Also cluster on Azure and cluster state on AWS is not production ready either.

Reply:
Why do you say it's not production ready? I don't think practicality plays a role in reliability/robustness required to treat something as "production-ready".

Reply:
The goal of this document is to be useful not just something that works. Having two cloud accounts to run a cluster on one is not something one would do in production, unless one cloud lacks the functionality. In this case Azure has a functionality we need to implement it or suggest using local dir for assets.

Reply:
I think we should remove production config for AKS then until we implement Azure backend support.

Reply:
Or leave this PR as it is for now.

Reply:
What do you mean by "leave"? To keep it open?

I think we should remove the production configuration.